search/indexing_api.py

```diff
@@ -109,12 +110,16 @@ def index_program_enrolled_users(program_enrollments, chunk_size=100):
     Returns:
         int: Number of indexed items
     """
-    return _index_chunks(
+    num_indexed = _index_chunks(
         (serialize_program_enrolled_user(program_enrollment) for program_enrollment in program_enrollments),
         USER_DOC_TYPE,
         chunk_size=chunk_size,
     )
 
+    for program_enrollment in program_enrollments:
+        send_automatic_emails(program_enrollment)
```

Comment:
The intention is to have this be flexible enough to trigger on any change which happens in the future where the user becomes part of a search query, not just for new program enrollments. I'll refactor this so that the email send happens in tasks triggered by signals and not during `recreate_index`